src/pal/src/include/pal/thread.hpp

```diff
@@ -314,6 +314,10 @@ namespace CorUnix
         DWORD m_dwLwpId;
         pthread_t m_pthreadSelf;        
 
+#if HAVE_MACH_THREADS
+        mach_port_t m_machPortSelf;
```

Comment:
You're correct, I just checked and it isn't cloned or ref'd so this should be ok.